docs/performance/scaling.md

```diff
@@ -48,25 +48,21 @@ large result sets from CrateDB][fetching large result sets from cratedb].
 
 ## Number of shards
 
-In CrateDB data in tables and partitions is distributed in storage units that we
-call shards.
+In CrateDB data in tables and partitions is distributed in storage units
+called "shards".
 
-If we do not specify how many shards we want for a table/partition CrateDB will
+If we do not specify how many shards we want for a table/partition, CrateDB will
 derive a default from the number of nodes.
 
-CrateDB also has replicas of data and this results in additional shards in the
-cluster.
```

Comment:
Why do we remove this, isn't this interesting to note?

Reply:
This statement has been refactored to the "advanced page" to reduce cluttering of valuable pro-tip information across different pages.

-- https://cratedb-guide--355.org.readthedocs.build/performance/sharding.html#partitions